js/search.js

```diff
@@ -892,6 +892,13 @@
 				abilitySet.push(['header', "Special Event Ability"]);
 				abilitySet.push(['ability', toId(template.abilities['S'])]);
 			}
+			if (format === 'almostanyability' || format === 'balancedhackmons' || format === 'bh') {
+				abilitySet.push(['header', "Other Abilities"]);
```

Comment:
This is "All Abilities".

Reply:
It's not like you're intentionally skipping abilities the pokemon already has.

Also you're not going to filter out unviable abilities? Just filter out `score <= 0`.